packages/components/ContextualMenu/src/ContextualMenu.types.ts

```diff
@@ -0,0 +1,33 @@
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import { ICalloutProps, ICalloutTokens } from '@fluentui-react-native/callout';
+
+export const contextualMenuName = 'ContexualMenu';
+
+export type IContextualMenuTokens = ICalloutTokens;
+/**
+ * Properties and Tokens for FluentUI React Native ContextualMenu
+ */
+
+export interface IContextualMenuProps extends Omit<ICalloutProps, 'setInitialFocus'> {
+  /*
+  * Whether to set initial focus on the contextual menu container, as opposed to the first menu item.
+  */
+  shouldFocusOnContainer?: boolean;
+
+  /*
+  * Whether tofocus on the menu when mounted
+  */
+  shouldFocusOnMount?: boolean;
+}
+
+export type IContextualMenuSlotProps = {
+  root: ICalloutProps;
+};
+
+export type IContextualMenuRenderData = IRenderData<IContextualMenuSlotProps>;
```

Comment:
I may have missed something, but I don't see usage of this type anywhere, we can air on the side of at least not exporting, but preferably get rid of it.